code/modules/research/xenobiology/vatgrowing/samples/_micro_organism.dm

```diff
@@ -0,0 +1,123 @@
+///A single type of growth.
+/datum/micro_organism
+	///Name, shown on microscope
+	var/name = "Unknown fluids"
+	///Desc, shown by science goggles
+	var/desc = "White fluid that tastes like salty coins and milk"
+
+///A "mob" cell. Can grow into a mob in a growing vat.
+/datum/micro_organism/cell_line
+	///Our growth so far, needs to get up to 100
+	var/growth = 0
+	///All the reagents required for letting this organism grow into whatever it should become
+	var/list/required_reagents
+	///Reagents that further speed up growth, but aren't needed.  Assoc list of reagent datum || bonus growth per tick
+	var/list/supplementary_reagents
+	///Reagents that surpress growth. Assoc list of reagent datum || lost growth per tick
+	var/list/suppressive_reagents
```

Comment:
Make it clear if you are referencing type paths, singletons or unique instantiated atoms that need to be cleaned